x/staking/migrations/v5/store.go

```diff
@@ -0,0 +1,27 @@
+package v5
+
+import (
+	storetypes "cosmossdk.io/store/types"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/staking/types"
+)
+
+func MigrateStore(ctx sdk.Context, storeKey storetypes.StoreKey, cdc codec.BinaryCodec) (storetypes.KVStore, error) {
```

Comment:
it would be good if we had some tooling to grab a state export from any mainnet then run the migration to see what the time will be. This would allow us to add warnings for users when upgrading. 

Out of scope for this pr but something we should so in the near future